content/docs/sampling.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Sampling
+rank: 8
```

Comment:
I think Sampling should be next to Client Libraries.

Reply:
Okay I'll re-adjust the ranking.